build_tools/docker/py38.dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM python:3.8.16-bullseye
+FROM python:3.8.18-slim-bookworm
```

Comment:
Likewise, we might have another security patch before its EOL (later this year), so it'll be a good idea to exclude patch version 18 from the docker image.